openquake/commonlib/config.py

```diff
@@ -68,17 +78,50 @@ def set(self, name, dic):
 
     def _get_paths(self):
         """Return the paths for the global/local configuration files."""
-        global_path = os.environ.get("OQ_SITE_CFG_PATH", self.GLOBAL_PATH)
-        local_path = os.environ.get(
-            "OQ_LOCAL_CFG_PATH", os.path.abspath(self.LOCAL_PATH))
-        paths = [global_path, local_path]
-
-        # User specified
-        user_path = os.environ.get(OQ_CONFIG_FILE_VAR)
-        if user_path is not None:
-            paths.append(user_path)
+        if sys.platform == 'win32':
+            local_prefix = os.environ['APPDATA']
+        elif sys.platform == 'darwin':
+            local_prefix = '~/Library/Preferences'
+        else:
+            local_prefix = os.environ.get('XDG_CONFIG_HOME', '~/.config')
+
+        local_path = os.path.join(local_prefix, self.CFG_PREFIX,
+                                  self.CFG_FILE)
+
+        paths = [local_path]
+
+        # when not running in a virtualenv system config
+        # has the highest priority
+        if not hasattr(sys, 'real_prefix'):
+            paths.append(os.path.join(self.SYS_PREFIX, self.CFG_PREFIX,
```

Comment:
It seems that 'real_prefix' does not work for people using `python3 -m venv` instead of virtualenv, see the discussion in http://stackoverflow.com/questions/1871549/python-determine-if-running-inside-virtualenv